### PR TITLE
Coarse loss weight 0.5 (test if over-regularizing)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The coarse loss is ESSENTIAL (removing = catastrophic), and weight 1.5 was neutral. But weight 0.5 hasn't been tested. If 1.0 is slightly over-regularizing, 0.5 might give the optimizer more freedom while keeping the essential structural signal.

## Instructions

Replace line 665:
```python
loss = loss + 1.0 * coarse_loss
```
With:
```python
loss = loss + 0.5 * coarse_loss
```

Run: `python train.py --agent norman --wandb_name "norman/coarse-0.5" --wandb_group coarse-weight-half`

## Baseline
- val/loss: 2.2068, surf_p: in_dist=20.56, tandem=40.78
---
## Results

**W&B run:** llxniu9u

| Metric | Baseline (w=1.0) | Coarse-0.5 | Δ |
|--------|-----------------|------------|---|
| val/loss | 2.2068 | 2.2293 | +0.0225 (+1.0%) |
| in_dist surf_p | 20.56 | 20.41 | **-0.15** |
| tandem surf_p | 40.78 | 41.83 | +1.05 |
| ood_cond surf_p | — | 20.64 | — |
| ood_re surf_p | — | 30.63 | — |

**Full surface MAE:**
- in_dist: Ux=0.286, Uy=0.175, p=20.41
- tandem: Ux=0.622, Uy=0.333, p=41.83
- ood_cond: Ux=0.259, Uy=0.186, p=20.64
- ood_re: Ux=0.272, Uy=0.197, p=30.63

**Full volume MAE:**
- in_dist: Ux=1.333, Uy=0.467, p=26.65
- tandem: Ux=2.162, Uy=1.009, p=45.04
- ood_cond: Ux=1.063, Uy=0.411, p=20.06
- ood_re: Ux=1.060, Uy=0.444, p=51.53

**Peak memory:** not logged

### What happened
Mixed results. In_dist surf_p improves slightly (20.41 vs 20.56, -0.15), suggesting weaker coarse regularization helps in-distribution generalization. However, tandem transfer gets worse (+1.05), which drives the overall val/loss higher (+1.0%). The coarse loss appears to provide structural regularization that particularly benefits tandem cases — reducing it from 1.0 to 0.5 slightly under-regularizes on the harder generalization task. The in_dist improvement is likely within run variance.

### Suggested follow-ups
- Try coarse weight 0.75 as a midpoint between 0.5 and 1.0 — might capture the in_dist benefit without tandem regression
- The current weight of 1.0 appears to be a reasonable sweet spot; this direction doesn't look promising for large gains

### CODE REVIEW

**Comment by tcapelle** [comment]
## Review — Closing
Essentially neutral (within noise of baseline). Closing.
